securedrop/tests/functional/journalist_navigation_steps.py

```diff
@@ -650,7 +666,9 @@ def _set_hotp_secret(self):
         submit_button.click()
 
     def _visit_edit_totp_secret(self):
-        self._visit_edit_secret('totp')
+        self._visit_edit_secret(
+            'totp',
+            'Reset 2FA for mobile apps such as FreeOTP or Google Authenticator')
```

Comment:
Functional tests may be run against locale settings other than English, so the string to check should be gettext()ed.